<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PagoWindow</class>
 <widget class="QWidget" name="PagoWindow">
  <property name="windowTitle">
   <string>Pago</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget {
        background-color: #e6f2ff;
        font-family: Arial;
        font-size: 14px;
        color: #000000;
    }
    QLabel {
        color: #005c99;
        font-weight: bold;
    }
    QLineEdit {
        border: 1px solid #80bfff;
        border-radius: 5px;
        padding: 5px;
        background-color: white;
        color: #000000;
    }
    QPushButton {
        background-color: #00cc99;
        color: white;
        border-radius: 10px;
        padding: 8px;
    }
    QPushButton:hover {
        background-color: #009973;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelPrecio">
     <property name="text">
      <string>Total: 0.00 €</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackTarjeta">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageTarjeta">
      <layout class="QVBoxLayout" name="verticalLayoutTarjeta">
       <item>
        <widget class="QLabel" name="labelNumero">
         <property name="text">
          <string>Número de Tarjeta</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputNumero"/>
       </item>
       <item>
        <widget class="QLabel" name="labelCaducidad">
         <property name="text">
          <string>Fecha de Caducidad</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputCaducidad"/>
       </item>
       <item>
        <widget class="QLabel" name="labelCVV">
         <property name="text">
          <string>CVV</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputCVV"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnPagar">
     <property name="text">
      <string>Pagar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
